android/sample/src/debug/java/com/facebook/flipper/sample/ExampleActions.java

```diff
@@ -22,66 +22,72 @@
 
 public final class ExampleActions {
 
-  public static void sendPostRequest(OkHttpClient client) {
-    final RequestBody formBody =
-        new FormBody.Builder().add("app", "Flipper").add("remarks", "Its awesome").build();
-
-    final Request request =
-        new Request.Builder()
-            .url("https://demo9512366.mockable.io/SonarPost")
-            .post(formBody)
+    // Create a custom OkHttpClient with timeouts
+    private static final OkHttpClient client = new OkHttpClient.Builder()
+            .connectTimeout(15, TimeUnit.SECONDS)
+            .readTimeout(15, TimeUnit.SECONDS)
             .build();
 
-    client
-        .newCall(request)
-        .enqueue(
-            new Callback() {
-              @Override
-              public void onFailure(final Call call, final IOException e) {
-                e.printStackTrace();
-                Log.d("Flipper", e.getMessage());
-              }
+    public static void sendPostRequest() {
```

Comment:
https://github.com/facebook/flipper/blob/5f9000aa82cfb1ce77576d22ac608356263cd393/android/sample/src/main/java/com/facebook/flipper/sample/RootComponentSpec.java#L140

This gets called here with a client, so I would expect that call to fail now. In general, having a client as a static field does not seem great to me as it will stay in memory longer than we need it.

Reply:
made the changes as mentioned kindly review an let me know